kubernetes-client-api/src/main/java/io/fabric8/kubernetes/client/internal/CertUtils.java

```diff
@@ -196,6 +204,8 @@ public PrivateKey call() {
     } catch (NoClassDefFoundError e) {
       throw new KubernetesClientException(
           "JcaPEMKeyConverter is provided by BouncyCastle, an optional dependency. To use support for EC Keys you must explicitly add this dependency to classpath.");
+    } catch (NullPointerException npe) {
+      throw new KubernetesClientException("Could not load EC key.");
```

Comment:
Where is the NPE coming from? It would probably be better to catch it closer to where it is happening, or put a null check.